src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/query/AbstractQueryStreamResult.java

```diff
@@ -51,6 +51,10 @@ public abstract class AbstractQueryStreamResult extends AbstractPayload implemen
     protected ObTableQuery                                                     tableQuery;
     protected long                                                             operationTimeout    = -1;
     protected String                                                           tableName;
+    // use to store the TableEntry Key:
+    // primary index or local index: key is primary table name
+    // global index: key is index table name (be like: __idx_<data_table_id>_<index_name>)
```

Comment:
Could we use TableName to carry this information?

Reply:
We couldn't. TableName will be as a part of the QueryRequest, but indexTableName is only used as a key of table entry map.